docs/setup-steps-1.4.0.md

```diff
@@ -0,0 +1,265 @@
+       ```
+
+       For real MDS/SBI, the vendors can download the new encryption key from the UI and test with the updated **SBI** which uses this encryption key. It can be downloaded for **Auth SBI** projects from UI.
+
+       ![](\_images/ctk-encryptionkey.png)
+       
+6. Ensure that `reporting` module is deployed from the `develop` branch. This is required for the **Kibana Dashboard**.
+
+7. Ensure that the `kernel-auth-adapter-1.2.0.1` has been successfully deployed in the `compliance-toolkit-service`, and verify that the identical authentication adapter was utilized to configure the mock ABIS.
+
+8. Check datashare configurations for **ABIS3030** and **ABIS3031** testcases.
+
+    * Ensure that in `data-share-default.properties` and `compliance-toolkit-default.properties`, the value of `mosip-abis-client` is set in `auth.server.admin.allowed.audience`. If this was not set by default, then set it and restart `data-share-service` and `compliance-toolkit-service`.
+
+    * For handle CTK datashare token flow, add the below values in `data-share-default.properties`.
+
+        ```
+        auth.handle.ctk.flow=true
+        mosip.api.internal.toolkit.url=https://${mosip.api.internal.host}/v1/toolkit
+        mosip.compliance.toolkit.saveDataShareToken.url=${mosip.api.internal.toolkit.url}/saveDataShareToken
+        mosip.compliance.toolkit.invalidateDataShareToken.url=${mosip.api.internal.toolkit.url}/invalidateDataShareToken
+        mosip.compliance.toolkit.invalidateDataShareToken.testCaseId=ABIS3031
+        ```
+
+
+### Steps to load testdata, schemas in MINIO
+
+1\. Browse [mosip-compliance-toolkit](https://github.com/mosip-compliance-toolkit.git)
+
+2\. Project structure will be as shown below.
+
+![](\_images/ctk-project-structure.png)
+
+3\. The resources folder has schemas and test data that need to be added to MinIO.
+
+![](\_images/ctk-resources-folder-new.png)
+
+#### Steps to load testdata and schemas directly in MinIO
+
+1\. Log in to MinIO from the browser.
+
+2\. Create a `compliance-toolkit` bucket.
+
+3\. Create a new folder named `testdata` in the above bucket.Upload MOSIP\_DEFAULT\_XXX.zip files from resources to it.
+
+4\. Create a new folder named `schemas` in the above bucket. Upload all **SBI**, **SDK** and **ABIS** schemas along with subfolders in it.
+
+5\. Upload `testcase_schema.json` from resources folder to `schemas` folder.
+
+6\. Please restart the compliance pods after adding new files in minio to refresh the cache.
+
+**Alternately, swagger endpoint can also be used to upload data in Minio. In this case there is no need to restart CTK services.**
+
+#### Steps to upload resources to Minio using Swagger
+
+1\. The swagger url is:
+
+`https://{api-internal-env-url}/v1/toolkit/swagger-ui/index.html?configUrl=/v1/toolkit/v3/api-docs/swagger-config`
+
+2\. Using keycloak/ register option in CTK UI, create a new user for compliance toolkit.
```

Comment:
Using keycloak / register option in CTK UI, create a new user for compliance toolkit.